### DISCUSSION TITLE
Trying to connect nominatim to the external postgres DB

### DISCUSSION THREAD

**Discussion by MALKARAJ** [mediagis/nominatim-docker#362] [Q&A] [answered]
HI Team,
I am getting this error while trying to connect nominatim with external data . kindly give me some hint about this error

![Screenshot from 2022-08-21 23-04-06](https://user-images.githubusercontent.com/62066377/185803868-997b6778-586f-429d-9799-522fa988a278.png)

**Comment by philipkozeny** [comment, marked as answer]
You set the address to `127.0.0.1` in the run command, which is the localhost inside the docker container, so its the internal DB and not external.
